easybuild/easyblocks/t/tensorflow.py

```diff
@@ -112,25 +113,47 @@ def configure_step(self):
                 filtered_path = os.pathsep.join([p for fil in path_filter for p in path if fil not in p])
                 env.setvar(var, filtered_path)
 
+        use_mpi = self.toolchain.options.get('usempi', False)
+
         # put wrapper for Intel C compiler in place (required to make sure license server is found)
         # cfr. https://github.com/bazelbuild/bazel/issues/663
         if self.toolchain.comp_family() == toolchain.INTELCOMP:
-            wrapper_dir = os.path.join(tmpdir, 'bin')
+            icc_wrapper_dir = os.path.join(tmpdir, 'bin')
 
             icc_wrapper_txt = INTEL_COMPILER_WRAPPER % {
                 'compiler_path': which('icc'),
+                'intel_mpi_root': '',
                 'cpath': os.getenv('CPATH'),
                 'intel_license_file': os.getenv('INTEL_LICENSE_FILE', os.getenv('LM_LICENSE_FILE')),
-                'wrapper_dir': wrapper_dir,
+                'wrapper_dir': icc_wrapper_dir,
             }
-            icc_wrapper = os.path.join(wrapper_dir, 'icc')
+            icc_wrapper = os.path.join(icc_wrapper_dir, 'icc')
             write_file(icc_wrapper, icc_wrapper_txt)
-            env.setvar('PATH', os.pathsep.join([os.path.dirname(icc_wrapper), os.getenv('PATH')]))
+            env.setvar('PATH', os.pathsep.join([icc_wrapper_dir, os.getenv('PATH')]))
+
+            if use_mpi:
+                mpiicc_wrapper_dir = os.path.join(tmpdir, 'bin2')
+                mpiicc_wrapper_txt = INTEL_COMPILER_WRAPPER % {
+                    'compiler_path': which('mpiicc'),
+                    'intel_mpi_root': os.getenv('I_MPI_ROOT'),
+                    'cpath': os.getenv('CPATH'),
+                    'intel_license_file': os.getenv('INTEL_LICENSE_FILE', os.getenv('LM_LICENSE_FILE')),
```

Comment:
Since we use these in two places, we should grab them only once from the environment above:

```python
wrapper_cpath = os.getenv('CPATH')
wrapper_intel_licfile = os.getenv('INTEL_LICENSE_FILE', os.getenv('LM_LICENSE_FILE'))